next.config.js

```diff
@@ -15,12 +15,22 @@ module.exports = (phase, { defaultConfig }) => {
   if (process.env.API_BASE_URL === undefined) {
     require('dotenv').config({path: '.env.build'})
   }
+  // Assign all values from .env file here.
+  // NOTE: If any values are added to .env, they must be applied here.
+  // TODO: Find a better way to handle this.
```

Comment:
You could make an array of all these variables and then loop over it.

Reply:
I think there are some changes to how environment variables are handled with next.js/vercel, so we should investigate that for a future PR.